app/services/api/webflow-helpers.ts

```diff
@@ -0,0 +1,182 @@
+import { Schedule } from "../../screens"
+import { ScheduleCardProps } from "../../screens/ScheduleScreen/ScheduleCard"
+import { useSchedule, useWorkshops } from "./webflow-api"
+import {
+  CustomScheduleProps,
+  CustomSpeakerProps,
+  CustomWorkshopProps,
+  ScheduleProps,
+  SpeakerProps,
+  WorkshopProps,
+} from "./webflow-api.types"
+
+/*
+ * Converting workshop data from "type ids" to "type names"
+ */
+export const cleanedWorkshops = (
+  workshopsData?: CustomWorkshopProps[],
+  speakersData?: SpeakerProps[],
+): WorkshopProps[] => {
+  return workshopsData
+    ?.filter((workshop) => !workshop._archived && !workshop._draft)
+    .map((workshop) => ({
+      ...workshop,
+      level:
```

Comment:
Oops, blew by this section first, see map pattern below that could be used here also instead of a bunch of ternary